lib/wallet/txdb.js

```diff
@@ -131,6 +145,17 @@ class TXDB {
     return this.removeOutpointMap(b, coin.hash, coin.index);
   }
 
+  /**
+   * Remove spent credit.
+   * @param {Credit} credit
+   * @param {Path} path
+   */
+
+  removeUnpentCredit(b, credit, path) {
```

Comment:
missing the 's' in "unspent"... but I think we might want to rename these two functions anyway:

So maybe instead of 
```
  saveUnspentCredit(b, credit, path) {
    const {coin} = credit;
```

we do:
```
await this.removeUnspentCoin(b, credit.coin, path);
```